eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -99,5 +99,17 @@ func (c *ClusterLoader) buildConfigMapData() (map[string]string, error) {
 }
 
 func (c *ClusterLoader) buildArgs() string {
-	return fmt.Sprintf("[%s]", strings.Join(c.Config.Spec.ClusterLoader.TestParams, ","))
+	return fmt.Sprintf("[%s]", strings.Join(append(c.Config.Spec.ClusterLoader.TestParams, "--report-dir=/usr/share/cluster-loader"), ","))
```

Comment:
nit: What led you to use /usr/share? I might use something like /var/reports.
https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/var.html

Reply:
Maybe I had some weird testing configurations, but I thought only /usr/share had worked. Now that I know a little more, I can see if /var/reports would work.